src/components/timeline/TimelineMarkerCanvas.js

```diff
@@ -179,11 +173,16 @@ class TimelineMarkerCanvas extends PureComponent {
               }
             }
           } else {
+            const radiusRatio = w / h;
+            // TODO Should we use an exponential scale instead ?
+            const radius =
+              radiusRatio * (MARKER_MAX_DOT_RADIUS - MARKER_MIN_DOT_RADIUS) +
+              MARKER_MIN_DOT_RADIUS;
             ctx.beginPath();
             ctx.arc(
-              x + w / 2, // x
```

Comment:
This along with the minimum value `10` for `w` was translating the dots to the right which makes them show misaligned with rectangles (easy to see with `performance.measure()` like in http://localhost:4242/public/adf5936ceaacb7f949a0e361d789e3fb52746cba/timeline/?hiddenThreads=&range=1.1999_2.8025&react_perf&thread=0&threadOrder=0-2-3-4-5-6-1 (look for `my-marker` and its siblings dots `my-marker-start` and `my-marker-end` in Content 5)